### PR TITLE
Jasmine test helpers

### PR DESCRIPTION
**Pull request by lnagle93**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
         "eslint": "^2.10.2",
         "istanbul": "^0.4.2",
         "jasmine-node": "^1.14.5",
+        "jasmine-node-helpers": "https://github.com/tests-always-included/jasmine-node-helpers",
```
Comment (fidian) [inline package.json]: We should wait until the repo is pushed to npm and renamed.  After that, this line should simply be

    "jasmine-test-helpers": "^1.0.0",

Or whatever the current version is.

Comment (lnagle93) [inline package.json]: Sounds good. I'll make that change.

**Comment by AbsentSemicolon** [comment]
Looks good to me once we get the package published and can include it.

**Comment by fidian** [comment]
Approved.  Squashing and merging.

**Comment by fidian** [comment]
The one test is failing because of an issue introduced into Node.  https://github.com/nodejs/node/issues/8571

**Pull request merged**